src/ape_ethereum/ecosystem.py

```diff
@@ -161,7 +162,9 @@ def get_proxy_info(self, address: AddressType) -> Optional[ProxyInfo]:
                 target = self.conversion_manager.convert(match.group(1), AddressType)
                 return ProxyInfo(type=type, target=target)
 
-        str_to_slot = lambda text: int(keccak(text=text).hex(), 16)  # noqa: E731
```

Comment:
Is this because it returns a pointer here that you turn this into a method? Just curious if you were running into issues

Reply:
Lambdas should only be used as anonymous functions. When you assign it to a variable, it isn' really anonymous so you might as well use a `def` statement. There is some more info here: https://www.flake8rules.com/rules/E731.html

The `#noqa` statement allowed it to go uncaught.